torch/sparse/__init__.py

```diff
@@ -357,3 +358,102 @@ def sum(input: Tensor, dim: DimOrDims = None,
             [0, 0, 0, 0, 0],
             [0, 0, 0, 0, 0]])
 """)
+
+
+class check_sparse_tensor_invariants(object):
+    """A tool to control checking sparse tensor invariants.
+
+The following options exists to manage sparsr tensor invariants
+checking in sparse tensor construction:
+
+1. Using a context manager::
+
+     with torch.sparse.check_sparse_tensor_invariants(enable=True):
```

Comment:
nit: This makes it look like you have to set `enable=True`, but actually it is set to `True` by default already. It makes the docs look a bit saner if the user doesn't have to repeat their intent with the API call or put differently, we don't have `enable=False` by default.

Reply:
Fixed.